docker-compose.yaml

```
@@ -30,8 +44,54 @@ services:
+        condition: service_healthy
+      minio:
+        condition: service_started
+    networks:
+      - shared_network
 volumes:
   pg_data:
   redis_data:
   minio_data:
+networks:
+  shared_network:
+    external: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)